src/main/java/com/xpanxion/java/springboot/da1/demo/model/student1/WorkoutHistory1.java

```diff
@@ -0,0 +1,63 @@
+package com.xpanxion.java.springboot.da1.demo.model.student1;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name = "workoutHistory1")
+public class WorkoutHistory1 {
+
+    @Id
+    @SequenceGenerator(
+            name = "student1_workoutHistorySequence",
+            sequenceName = "student1_workoutHistorySequence",
+            allocationSize = 1
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "student1_workoutHistorySequence"
+    )
+    private int id;
+    private int memberId;
+    private Date timeUtc;
+    private String checkType;
```

Comment:
This could be tightened up a bit by using an enum, something like:
```
public enum CheckType {
    CHECK_IN, 
    CHECK_OUT 
}
```

Reply:
ah, I forgot all about enum's use cases. Thanks, I'll make this change